WebRoot/js/zimbraMail/prefs/view/ZmFilterRulesView.js

```diff
@@ -35,18 +35,15 @@
  */
 ZmFilterRulesView = function(parent, controller) {
 
-	DwtTabViewPage.call(this, parent, "ZmPreferencesPage ZmFilterRulesView", Dwt.STATIC_STYLE);
+	DwtTabViewPage.call(this, parent, "ZmFilterRulesView", Dwt.STATIC_STYLE);
 
 	this._controller = controller;
 	this._prefsController = AjxDispatcher.run("GetPrefController");
 
-	this._rules = AjxDispatcher.run(controller.isOutgoing() ? "GetOutgoingFilterRules" : "GetFilterRules");
```

Comment:
This change listener has been moved to https://github.com/Zimbra/zm-web-client/pull/123/files/d8d573a07657e66d436b51f5397758620d661f22#diff-ada83781749503b4db7ab2bcf7361113R190

It was there initially but not needed now.